dicom-archive/updateMRI_Upload.pl

```diff
@@ -3,6 +3,87 @@
 # zia.mohades@mcgill.ca
 # Perl tool to update the mri_upload table
 
+=pod
+
+=head1 NAME
+
+updateMRI_Upload.pl - updates database table mri_upload according to an entry in table
+   tarchive
+
+=head1 SYNOPSIS
+
+updateMRI_Upload.pl [options] -profile prod -tarchivePath tarchivePath -source_location source_location
+
+=over 2
+
+=item *
+B<-profile prod> : (mandatory) path (absolute or relative to the current directory) of the 
+    profile file
+
+=item *
+B<-tarchivePath tarchivePath> : (mandatory) absolute path to the tarchive file
+
+=item *
+B<-source_location source_location> : (mandatory) value to set column 
+    DecompressedLocation to for the newly created record in table mri_upload (see below)
+    
+=item *
+B<-globLocation> : Loosen the validity check of the tarchive allowing for the 
+     possibility that the tarchive was moved to a different directory.
+
+=item *
+B<-verbose> : Be verbose
+
+=back 
+
+=head1 DESCRIPTION
+
+This script first starts by reading the F<prod> file (argument passed to the C<-profile> switch)
+to fetch the C<@db> variable, a Perl array containing four elements: the database
+name, the databse user name used to connect to the database, the password and the 
+database hostname. It then checks for an entry in table C<tarchive> with the same 
```

Comment:
replace "in table `tarchive`" by "in the `tarchive` table"

Reply:
done